CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.13.1
+* Enhancement - Added cache for DiscoverPollEndPoint API.
+* Enhancement - Expose port 51679 so docker tasks can fetch IAM credentials.
+* Bug - fixed a bug that could lead to exhausting the open file limit.
+* Bug - Fixed a bug where images were not deleted when using image cleanup.
+* Bug - Fixed a bug where task status may be reported as pending while task is running.
+* Bug - Fixed a bug where task may have a temporary "RUNNING" state when
+  task failed to start.
+* Bug - Fixed a bug where CPU metrics would be reported incorrectly for some kernels.
```

Comment:
Can you modify this to be more specific about which kernel version would be affected by this?